src/unistd.rs

```diff
@@ -735,11 +735,21 @@ pub fn fchown(fd: RawFd, owner: Option<Uid>, group: Option<Gid>) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
-/// Flags for `fchownat` function.
-#[derive(Clone, Copy, Debug)]
-pub enum FchownatFlags {
-    FollowSymlink,
-    NoFollowSymlink,
+libc_bitflags! {
+    /// Flags for the `fchownat` function.
+    pub struct FchownatFlags: c_int {
```

Comment:
For this type, I am not sure if we should define a new type or just use the [AtFlags](https://docs.rs/nix/latest/nix/fcntl/struct.AtFlags.html) type.

It is safer to define a new type so that we can prevent users from passing invalid arguments, but this will add our maintainence burgen. We are already using the AtFlags type for other functions like [fstatat](https://docs.rs/nix/latest/nix/sys/stat/fn.fstatat.html)

Reply:
If we're already using AtFlags for more than one function, I think we can use it here, too.

Reply:
Get it